app/src/test/java/com/github/geohunt/app/mocks/MockBounty.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.geohunt.app.mocks
+
+import com.github.geohunt.app.model.Bounty
+import com.github.geohunt.app.model.Location
+import java.time.LocalDateTime
+
+fun MockBounty(
```

Comment:
This function is never used, so we should remove it. We are using the MockBounty in the androidTest version, but not in the unit test version.